### PR TITLE
Clarify protocol port-range docs for #2065

### LINKED ISSUE #2065: Documentation about transport layer ports being IPv4 implies they are not IPv6
### Describe the bug

As a developer directly using the official documentation directly through a browser or via OSCAL-based software that provide hints via this documentation, I would like clear explanation of [`/system-security-plan/system-implementation/component/protocol/port-range` inline documentation](https://github.com/usnistgov/OSCAL/blob/4f02dac6f698efda387cc5f55bc99581eaf494b6/src/metaschema/oscal_implementation-common_metaschema.xml#L280-L332) that suggests the poort range therein is "[w]here applicable this is the IPv4 port range on which the service operates." This framing implies the transport layer ports embedded with in IP (TCP; UDP; SCTP; et cetera) work with IPv4, but not IPv6. I would propose a change to tighten this wording.

### Who is the bug affecting

Developers who rely on documentation through direct review or prepare software the use this documentation to hint to developers proper definition of OSCAL data who want clear guidance on port and protocol ranges.

### What is affected by this bug

Documentation, Metaschema

### How do we replicate this issue

Review [the above Metaschema module reference](https://github.com/usnistgov/OSCAL/blob/4f02dac6f698efda387cc5f55bc99581eaf494b6/src/metaschema/oscal_implementation-common_metaschema.xml#L280-L332) and [resulting website documentation rendered](https://pages.nist.gov/OSCAL-Reference/models/v1.1.2/system-security-plan/json-reference/#/system-security-plan/system-implementation/components/protocols/port-ranges).

### Expected behavior (i.e. solution)

The documentation does not imply that port ranges are not IPv4 only, clarify they are transport layer protocols (e.g. SCTP, TCP, UDP).

### Other comments

_No response_

### Revisions

_No response_

### PR DESCRIPTION
**Pull request by aj-stein-gsa**
# Committer Notes

This documentation fix clarifies ambiguity about how transport layer ports conform with standards that layer on top of the Internet Protocol (IPv4, IPv6 for now, potentially others in the future), not directly as only as "IPv4 port ranges" as it implies.

Closes #2065.

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/OSCAL/blob/main/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Do all automated CI/CD checks pass?

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

(For reviewers: [The wiki has guidance](https://github.com/usnistgov/OSCAL/wiki/Issue-Review) on code review and overall issue review for completeness.)

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- ~Have you written new tests for your core changes, as applicable?~ This repo does not contain model-based/instance-based testing; we will update the GSA FedRAMP Automation Team's test suite accordingly for public review and ongoing use.
- ~Have you included examples of how to use your new feature(s)?~ Not in this repository, but will work with GSA FedRAMP Automation Team to update our constraints and examples accordingly.
- ~Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.~

### CODE REVIEW

**Comment by iMichaela** [comment]
@aj-stein-gsa - Port range error has been fixed. The fix is already in the `develop` branch

**Comment by iMichaela** [comment]
@aj-stein-gsa - please submit the PR against the `develop` branch so we can expedite the patch release once reviewed.

**Comment by aj-stein-gsa** [comment]
> @aj-stein-gsa - please submit the PR against the `develop` branch so we can expedite the patch release once reviewed.

Did you read explicitly what I wrote in the PR message and the related mailing list discussion about hotfix and following NIST OSCAL release guidance? It may seem redundant, but it is important. If you are not willing to accept this PR and accept my proposed plan, please me know. I will close this PR and plan accordingly.

**Comment by aj-stein-gsa** [comment]
@iMichaela, against the documented guidance and my recommendations, per your request I rebased and this PR now commits to develop. I would welcome the following.

1. Enable the GitHub Actions workflow to run the CI/CD actions and validate it is good for merge, post-approval.
2. A review to approve.

File: src/metaschema/oscal_implementation-common_metaschema.xml [hunk 1]
```diff
@@ -279,17 +279,17 @@
       </define-assembly>
       <define-assembly name="port-range">
             <formal-name>Port Range</formal-name>
-            <description>Where applicable this is the IPv4 port range on which the service operates.</description>
-            <define-flag name="start" as-type="non-negative-integer">
+            <description>Where applicable this is the transport layer protocol port range an IPv4-based or IPv6-based service uses.</description>
+            <define-flag name="start" as-type="nonNegativeInteger">
```
Comment (iMichaela) [inline src/metaschema/oscal_implementation-common_metaschema.xml:283]: Please update datatype `nonNegativeInteger` to `non-negative-integer`. We replaced the CamelCase datatypes.

**Review by iMichaela — CHANGES_REQUESTED** [review]

File: src/metaschema/oscal_implementation-common_metaschema.xml (hunk 1 above)
Comment (aj-stein-gsa) [reply src/metaschema/oscal_implementation-common_metaschema.xml]: Reverted in https://github.com/usnistgov/OSCAL/pull/2070/commits/415600ab6fb5d8d10ff63e1a10ac2d7eb58712ac.

**Comment by iMichaela** [comment]
@aj-stein-gsa - Thank you.

**Review by iMichaela — APPROVED** [review]
LGTM

**Pull request merged**